event.php

```diff
@@ -1228,22 +1228,27 @@ function updateEvent()
     $event->metaurl = $_POST['metaurl'];
     $event->reporturl = $_POST['reporturl'];
 
-    if ($_POST['mainrounds'] == '') {
-        $_POST['mainrounds'] = 3;
-    }
-    if ($_POST['mainstruct'] == '') {
-        $_POST['mainstruct'] = 'Swiss';
-    }
-    $event->mainrounds = $_POST['mainrounds'];
-    $event->mainstruct = $_POST['mainstruct'];
-    if ($_POST['finalrounds'] == '') {
-        $_POST['finalrounds'] = 0;
-    }
-    if ($_POST['finalstruct'] == '') {
-        $_POST['finalstruct'] = 'Single Elimination';
+    if ($event->current_structure !== 'Single Elimination') {
```

Comment:
What is the purpose of this?

Reply:
From what I see, if we change any number of rounds during single elimination rounds, it would break the code. So this code will make sure changes are only done during swiss round